src/java/picard/sam/FastqToSam.java

```diff
@@ -102,6 +102,12 @@
     @Option(shortName = "PI", doc = "Predicted median insert size, to insert into the read group header", optional = true)
     public Integer PREDICTED_INSERT_SIZE;
 
+    @Option(shortName = "PG", doc = "Program used for processing the read group.", optional=true)
+    public String PROGRAM_GROUP;
+
+    @Option(shortName = "PM", doc = "Free-form text providing further details of the platform/technology used.", optional=true)
```

Comment:
Perhaps work the phrase "platform model" into the doc to justify the choice of shortName?  :)